spec/installer-spec.ts

```diff
@@ -38,4 +41,19 @@ test('creates a nuget package and installer', async (t): Promise<void> => {
 
   log('Verifying Update.exe');
   t.true(await fs.pathExists(path.join(appDirectory, 'Squirrel.exe')));
+
+  log('Verifying contents of .nupkg');
+  const sevenZipPath = path.join(__dirname, '..', 'vendor', '7z.exe');
+  let cmd = sevenZipPath;
+  const args = ['l', nupkgPath];
+
+  if (process.platform !== 'win32') {
+    args.unshift(cmd);
+    const wineExe = process.arch === 'x64' ? 'wine64' : 'wine';
+    cmd = wineExe;
+  }
+
+  const packageContents = await spawn(cmd, args);
+  t.true(packageContents.includes('lib\\net45\\vk_swiftshader_icd.json'));
+  t.true(packageContents.includes('lib\\net45\\swiftshader\\libEGL.dll'));
```

Comment:
It would be nice if there was a test that didn't add the swiftshader files as well, to prevent regressions.

Reply:
Agreed, I'll take a stab at it!

Reply:
I encountered an issue where the two tests caused a race condition trying to delete the `Squirrel.exe` file in the `beforeEach` so I tweaked the approach to always copying the fixture app directory to a temporary location before packaging.